### ISSUE TITLE
Urgent! JEM 2.2.3+ Backend Problem creating new venue!

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1208]
I can reproduce this issue (from 2 others) in forefox in Backend on a clean joomla 3.9.14 on php 7.3.11 with protostar template and JEM 2.2.3. On JEM 2.3.0 prerelease too!
I don't know since when this comes. (Because the use of creating new venues is not so needed, it is possible, that it is not yet found earlier.)
In backend I create a new venue, I insert a name, then I go to image and choose an image on my harddisk. Then I click "upload", I get the notification, that the upload is ok. But immediately the editvenue window closes and there's no new venue saved.
(meanwhile the solution is: Before choosing and uploading an image, you first have to save the new venue, after you can upload an image!)
BUT: THIS ist not a solution! URGENT!
EDIT: the same problem, when you want to create a new event with an image!

I added this issue in Egnarts94/jem-project too #82

**Comment by depebo** [comment]
@jojo12 so, the issue is adding image to new (i.e. not yet saved) venue or event?
Category is fine?

**Comment by jojo12** [comment]
@depebo: you're right: the category image upload has the same problem!

**Comment by ernst67** [comment]
Just tested and I have the same issue....:

After uploading a venue image the 'upload complete' notification appears. When I click OK the editvenue windows closes. No venue added to the DB. Selecting an existing image from the server does work.
Same issue for categories and events!  
Tested on JEM 2.2.3 / Joomla 3.9.8 / PHP 7.3.2 / Isis backend template

How on earth have we never heard about this when it's even on a stable release...??

**Comment by jojo12** [comment]
I think it is quite new, seems a change in firefox.

**Comment by ernst67** [comment]
Tested some more on Chrome, Internet Explorer and Edge. No problem in any of those browsers so it seems to be limited to Firefox.
Tested with Firefox in Safe Mode and the problem persists.

Then I tested frontend. The problem does not occur there, because the behaviour is different. After upload you don't get an "upload-complete" message. (I guess the actual upload is only done upon submitting the form)
